docs/docs/normal-mode/space-menu.md

```diff
@@ -26,6 +26,7 @@ Contextual actions are actions that are only applicable within a specific contex
 | Keybinding | Action                                                           |
 | ---------- | ---------------------------------------------------------------- |
 | `a`        | Add a new file/folder under the current path [^1]                |
+| `c`        | Copy current file to a new file                                  |
```

Comment:
We should use `y` since it stands for Copy at the moment.

Also, I think `Copy current file to a new path` is more clear.

Reply:
y to me means yank, put it on the clipboard for example. In most File Managers you can "copy" a file and paste it somewhere else.

I think of this copy as an immediate action, hence different.

What do you think?

Reply:
Make sense